src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/completion/IJavaCompletionParticipant.java

```diff
@@ -24,8 +24,8 @@
  * @author datho7561
  */
 public interface IJavaCompletionParticipant {
-	// Not used in liberty-tools-intellij:
-	// public static final ExtensionPointName<IJavaCompletionParticipant> EP_NAME = ExtensionPointName.create("open-liberty.intellij.javaCompletionParticipant");
+	 ExtensionPointName<IJavaCompletionParticipant> EP_NAME =
+			ExtensionPointName.create("open-liberty.intellij.javaCompletionParticipant");
```

Comment:
@aparnamichael @turkeylurkey just noticed this change here. The previous comment said this was not used in LTI. Has that changed moving forward?

Reply:
@TrevCraw I believe I added it during the lsp4mp4ij update. I didn't notice that comment. Should I remove it?

Reply:
@aparnamichael, the comment is being removed and the extension point is being enabled. I am not sure if that is the correct thing to do or not. Here is the original PR which added these two extension points to the old LSP4IJ integration feature branch: https://github.com/OpenLiberty/liberty-tools-intellij/pull/737/files

Here is where the extension points were initially commented out in main: https://github.com/OpenLiberty/liberty-tools-intellij/commit/8737311800f6882f961c86b177ed1e07764f1591. It looks like @turkeylurkey commented out these extension points in response to this comment from @mrglavas : https://github.com/OpenLiberty/liberty-tools-intellij/pull/745#pullrequestreview-2050580925

We need to determine if these extension points should still be disabled or if they are being used now.

Reply:
I think it is used by MicroProfileFaultToleranceCompletionParticipant. That was added in January so I'm not sure why Michael said it is not used. The comment says it is used for the `@Fallback` annotation so we would not notice if it were missing. We don't test for that. I think it is only used for this one class.

Reply:
@TrevCraw @turkeylurkey @aparnamichael The extension point for completion in LTI is [JavaCompletionDefinition](https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/internal/core/java/completion/JavaCompletionDefinition.java). So this is bound to bean class in LTI's `plugin.xml` and not the `IJavaCompletionParticipant` interface. This work was done to create a common code path for supporting MicroProfile and Jakarta EE. The extension point that exists on the interface in LSP4MP4iJ is indeed not used because it was replaced by this one.